src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1037,9 +1037,33 @@ public float CalculateHitpoints()
         return maxHitpoints;
     }
 
-    public float CalculateStorage()
+    public float GetNominalCapacity()
     {
-        return MicrobeInternalCalculations.CalculateCapacity(editedMicrobeOrganelles);
+        return editedMicrobeOrganelles.Sum(o => MicrobeInternalCalculations
+            .GetNominalCapacityForOrganelle(o.Upgrades, o.Definition));
+    }
+
+    public Dictionary<Compound, float> GetAdditionalCapacities()
+    {
+        var dict = new Dictionary<Compound, float>();
```

Comment:
```suggestion
        var capacities = new Dictionary<Compound, float>();
```

"dict" is not a descriptive variable name.